django/core/management/utils.py

```diff
@@ -75,3 +76,37 @@ def find_command(cmd, path=None, pathext=None):
             if os.path.isfile(fext):
                 return fext
     return None
+
+
+def parse_model_labels(labels, source=None):
```

Comment:
Maybe "parse_apps_and_model_labels"? I agree the function doesn't have a great separate of concerns (in particular, the "source" parameter name doesn't seem obvious) but I don't have any better suggestions right now.